### ISSUE TITLE
JSON parse error on empty object

### ISSUE THREAD

**Issue by v1adko** [mixpanel/mixpanel-unity#87]
Using 2.1 release.

![Screen Shot 2019-11-27 at 2 21 33 PM](https://user-images.githubusercontent.com/11582927/69723302-ec0ba300-1121-11ea-934e-da885f332655.png)

Stacktrace
```
ArgumentException: JSON parse error: Missing a comma or '}' after an object member.
UnityEngine.JsonUtility.FromJsonOverwrite (System.String json, System.Object objectToOverwrite) (at /Users/builduser/buildslave/unity/build/Modules/JSONSerialize/Public/JsonUtility.bindings.cs:56)
mixpanel.MixpanelStorage.get_SuperProperties () (at Library/PackageCache/com.mixpanel.unity@6a4610ca3c1506e6af126139a4cbcfceaaee9dee/Mixpanel/Storage.cs:167)
mixpanel.Mixpanel.Register (System.String key, mixpanel.Value value
```

This https://github.com/InceptionXR/mixpanel-unity/commit/d3449a8009961fa84b0520dc4e25b5aae84094e0 fixes the issue for us but I do not fully know what it affects or what causes it.

**Comment by patedit** [comment]
Did you upgrade from 2.0.0? Is this from the Unity Editor? It'd be great to know what is stored in your storage which you can access on mac on the following folder `~/Library/Application Support/company name/product name`. Thanks for bringing this up!

**Comment by v1adko** [comment]
Yes, upgraded from 2.0. I cleared everything when running, so it was an empty folder.

**Comment by RedHatJef** [comment]
Worker.cs will crash when reading malformed json stored locally on device, and the only fix as a mobile user is to delete the app completely and reinstall, which isn't a great user experience.  I've submitted a pull request with a fix for this.  https://github.com/mixpanel/mixpanel-unity/pull/102

**Comment by zihejia** [comment]
Closing this one now for housekeeping. Fixes have been merged into v2.2.0. Thanks @RedHatJef .
